tests/async-unit/test_rxnorm_tools.py

```
@@ -0,0 +1,717 @@
+    @pytest.mark.asyncio
+    async def test_handle_does_not_match_prose(self):
+        """Plain prose without an explicit tool call should be ignored."""
+        from fighthealthinsurance.chat.tools.rxnorm_tool import RxNormLookupTool
+
+        send_status = AsyncMock()
+        rxnorm = RxNormTools()
+        with patch.object(
+            rxnorm, "get_brands_and_generics", new_callable=AsyncMock
+        ) as mock_info:
+            tool = RxNormLookupTool(send_status, rxnorm_tools=rxnorm)
+            cleaned, ctx, handled = await tool.handle(
```

Ruff (0.15.12)

[warning] Unpacked variable `ctx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)